File: 백준/Bronze/2581. 소수/소수.py
m = int(input())
n = int(input())
list_prime = []

def is_prime(num):
    if num < 2:
        return False
    for i in range(2, num):
        if num % i == 0:
            return False
    return True

for x in range(m,n+1,1):
    if is_prime(x):
        list_prime.append(x)

if not list_prime:
    print(-1)
else:
    print(sum(list_prime))
    print(min(list_prime))

File: 백준/Bronze/2581. 소수/소수.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include <ctype.h>
#include <limits.h>

#define TRUE 1 
#define FALSE 0

int is_prime(int n){
    if (n < 2) return FALSE;
    if (n == 2) return TRUE;  
    for(int i = 2; i <= (int)sqrt(n); i++){
        if(n % i == 0)return FALSE;
    }
    return TRUE;
}

int main() {
    int min = -1;
    int sum = 0;

    int M, N;
    scanf("%d %d", &M, &N);

    for (int i = M; i <= N; i++) {
        if (is_prime(i)) {
            sum += i;
            if (min == -1)
                min = i;
        }
    }

    if(sum == 0){
        printf("-1\n");
    }else{
        printf("%d\n%d\n", sum, min);
    }


    return 0;
}